deploy/roles/beaker/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+# main.yml - Beaker installation
+---
+- name: Ensure beaker directory is available
+  file:
+    path: "{{ beaker_src_dir }}"
+    state: directory
+    mode: 0755
+  tags:
+    - beaker
+
+- name: Download Beaker
+  get_url:
+    url: "{{ beaker_mirror }}"
+    dest: "{{ beaker_src_dir }}"
+  tags:
+    - beaker
+
+- name: Unarchive zipped beaker file
+  unarchive:
+    src: "{{ beaker_zip_file }}"
+    dest: "{{ beaker_parent_dir }}"
+    copy: no
+  tags: 
+    - beaker
```

Comment:
The "unarchive" command now supports downloading a remote archive and unzipping it all in one command, and I think this would be much cleaner here.  You would add a parameter "remote_src: yes", and remove "copy: no", then set src: to the Beaker URL itself.  This requires Ansible 2.2+, so you might need to update your ansible on the Mac.  If you did this, you could remove the two tasks before this one, and the "beaker_zip_file" and "beaker_src_dir" variables.